### PR TITLE
nicer looking action to create a new Kotlin file

### PR DESCRIPTION
**Pull request by yole**
#KT-8445 Fixed

### CODE REVIEW

File: idea/src/org/jetbrains/kotlin/idea/actions/NewKotlinFileAction.java [hunk 1]
```diff
@@ -55,8 +55,8 @@ protected void postProcess(PsiFile createdElement, String templateName, Map<Stri
     @Override
     protected void buildDialog(Project project, PsiDirectory directory, CreateFileFromTemplateDialog.Builder builder) {
         builder
-                .setTitle(JetBundle.message("new.kotlin.file.action"))
-                .addKind("Kotlin file", JetFileType.INSTANCE.getIcon(), "Kotlin File")
+                .setTitle("New Kotlin File/Class")
```
Comment (udalov) [inline idea/src/org/jetbrains/kotlin/idea/actions/NewKotlinFileAction.java]: Other stuff under the `New...` menu doesn't start with "New", this probably shouldn't start as well

Comment (yole) [inline idea/src/org/jetbrains/kotlin/idea/actions/NewKotlinFileAction.java]: This is the dialog title, not the name of the action in the menu. The dialog titles for Java and Groovy do have "New" in them.

**Comment by abreslav** [comment]
Looks good to me

**Comment by yole** [comment]
Merged manually.

**Pull request closed without merging**